Vagrantfile

```diff
@@ -64,10 +64,11 @@ Vagrant.configure("2") do |config|
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
   config.vm.provision "shell", inline: <<-SHELL
-     wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
+     wget --continue https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
      sudo dpkg -i erlang-solutions_1.0_all.deb
      sudo apt-get update
      sudo apt-get install -y esl-erlang=1:20.3.6
-     sudo apt-get install -y git
+     sudo apt-get install -y git make leiningen
+     sudo apt-get install -y openjdk-8-jre openjdk-8-jre-headless libjna-java
```

Comment:
Don't need Java.

Reply:
As per above comment ^ I'm probably doing it all wrong.